### PR TITLE
chore: create button atoms

### PR DESCRIPTION
**Pull request by theallegrarr**
### Description ?
A reusable button component has been created, it receives size, background color(optional) and font color(optional) as props, Stories were also set up to view variations of the component, and tests were added to check the component

#### How Has This Been Tested?
- Automated tests and manual checks

#### Checklist
- [x] 3 button types created(small,medium,large)
- [x] Stories created for the buttons and their props
- [x] Tests written for the buttons

### CODE REVIEW

**Comment by theallegrarr** [comment]
[Create Button atomic component](https://trello.com/c/1LmHJWnT/36-create-button-atomic-component)

File: components/atoms/ButtonAtoms/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function Buttons(props) {
+  const LargeButton = styled.button`
+    background-color: ${props => props.background || '#CE5374'};
```
Comment (twumm) [inline components/atoms/ButtonAtoms/index.js]: Considering props is already defined in the outer Buttons function, re-declaring props here will be repetition. 
One way to go about it is to remove the props from line 4

File: components/atoms/ButtonAtoms/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ButtonComponent from './index';
+
+export default { title: 'Button Component' };
+
+export const SmallButton = () => <ButtonComponent size="small" text="Small" />;
```
Comment (twumm) [inline components/atoms/ButtonAtoms/index.stories.js]: variables here can be camel case - `smallButton`.

**Review by twumm — CHANGES_REQUESTED** [review]
eslint had some sections marked up, can you kindly resolve those?

For the Button, we can keep the styles DRY by using props in this example below:
```
const Arrow = styled.i`
  border: solid #757575;
  border-width: 0 4px 4px 0;
  display: inline-block;
  padding: 6px;
  cursor: pointer;

  ${props => props.right && css`
    transform: rotate(-45deg);
    -webkit-transform: rotate(-45deg);
  `}
  ${props => props.left && css`
    transform: rotate(135deg);
    -webkit-transform: rotate(135deg);
    padding: 4px;
  `}
`;
```

In the above, the left and right arrows will inherit all styles for the regular arrow.

Please let me know if this is fine by you.
Perhaps by the time you get back, I will have had some other components you can checkout since the Heading component was quite unusual with me using `React.createElement`.

*Also*, you can delete the old Button folder so your folder can be named Button.

Cheers

**Review by twumm — APPROVED** [review]
Awesome work on the refactor! Please merge in.

**Pull request merged**